src/der.rs

```diff
@@ -168,6 +168,19 @@ pub fn small_nonnegative_integer(input: &mut untrusted::Reader)
     })
 }
 
+pub fn bit_string_with_no_unused_bits<'a>(input: &mut untrusted::Reader<'a>)
+                                          -> Result<untrusted::Input<'a>,
+                                              error::Unspecified> {
+    nested(input, Tag::BitString, error::Unspecified, |value| {
```

Comment:
Note to self: reformat this, both the return value and the `try!` indention below.